lib/generator.go

```diff
@@ -29,36 +29,71 @@ import (
 	"time"
 
 	"github.com/go-chef/chef"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
+	log "github.com/sirupsen/logrus"
 )
 
-func generateRandomData(chefClient chef.Client, requests amountOfRequests) (err error) {
-	channels := make([]<-chan error, config.NumNodes)
+func GenerateData(config *Config) error {
+	var amountOfRequests = make(amountOfRequests)
```

Comment:
This is a little confusing having the type `amountOfRequests` and the variable `amountOfRequests` have the same name.

Reply:
yeah agree